src/main/js/helpers/region.js

```diff
@@ -17,6 +17,70 @@ import utils from "./utils";
  * @alias module:region
  */
 
+/*
+ * global array to store the region if same
+ */
+const globalregionData = [];
+
+/**
+ * Compares the region to check is region is same for Line graph
+ * @private
+ * @param {Object} region - Region to be shown within graph
+ * @returns {Boolean} - returns true or false
+ */
+const compareRegionDataLine = (region) => {
+    if (globalregionData.length > 0) {
+        if (
+            globalregionData[0].start === region.start &&
+            globalregionData[0].end === region.end &&
+            globalregionData[0].axis === region.axis
+        ) {
+            return true;
+        } else {
```

Comment:
unnecessary use of else block, return directly

Reply:
isRegionSame me be set true for one load and for the second load the region is not same for line then we have to return false.

Reply:
Right, if it doesn't go into the if block, then simply return ```false``` would be sufficient. If it goes into if block, it returns ```true```

```
        if (
            globalregionData[0].start === region.start &&
            globalregionData[0].end === region.end &&
            globalregionData[0].axis === region.axis
        ) {
            return true;
        } else {
           return false;
       }   
```
is the same as 

```
        if (
            globalregionData[0].start === region.start &&
            globalregionData[0].end === region.end &&
            globalregionData[0].axis === region.axis
        ) {
            return true;
        } 
        return false;
```

Reply:
ok i will do the change